### PR TITLE
Progress

### PR DESCRIPTION
**Pull request by gintama91**
### Description
progress

### Image(if needed, helps for a faster review)

Screenshot_20231005_142038

### Checklist

- [ ] Run tests locally
- [ ] Run linter(check for linter errors)

### CODE REVIEW

File: lacci/lib/shoes/widgets/progress.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Shoes
+  class Progress < Shoes::Widget
```
Comment (noahgibbs) [inline lacci/lib/shoes/widgets/progress.rb:4]: Does it need a display property or something for % progress?

Comment (gintama91) [reply lacci/lib/shoes/widgets/progress.rb:4]: yeah it does i guess there is value and max as two proerties and
 i need to change this html into render method
 so i made this into draft

Comment (noahgibbs) [reply lacci/lib/shoes/widgets/progress.rb:4]: Ah, sorry, this says draft. So you're probably already on it. Looks like "fraction" is the Shoes style for it, and it's a float between 0.0 and 1.0.

**Comment by gintama91** [comment]
Screenshot_20231015_140238

**Comment by noahgibbs** [comment]
Nice! Love the screenshot.

**Review by noahgibbs — APPROVED** [review]

**Comment by gintama91** [comment]
I think this was not merged due to conflicts? so meging now as i resolved them.

**Comment by noahgibbs** [comment]
Yay! :tada:

**Pull request merged**